spring-boot-3-integrationtests/src/test/java/org/axonframework/eventsourcing/utils/EventStoreTestUtils.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2010-2023. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.eventsourcing.utils;
+
+import org.axonframework.common.IdentifierFactory;
+import org.axonframework.eventhandling.DomainEventMessage;
+import org.axonframework.eventhandling.GenericDomainEventMessage;
+import org.axonframework.messaging.MetaData;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public abstract class EventStoreTestUtils {
```

Comment:
A similar point as with the DLQ integration test suite applies here.
I think we can come by when we add this dependency so that we do not have to copy it over:
```xml
<dependency>
    <groupId>${project.groupId}</groupId>
    <artifactId>axon-eventsourcing</artifactId>
    <version>${project.version}</version>
    <classifier>tests</classifier>
    <scope>test</scope>
</dependency>
```